### ISSUE TITLE
Custom Tickets: support immutable properties 

### ISSUE THREAD

**Issue by sanmont3drepo** [3drepo/3drepo.io#4752]
### Description
We are adding the concept of "immutable properties".  Immutable are properties in a ticket template that are writable as long as they doesn't have a value. 

Example 1:
- property1 is immutable
- I set property1 in ticket creation
- When I open again the ticket property1 becomes read-only

Example 2:
- property2 is immutable
- I create the ticket without setting property2 
- When I open again the ticket property2 is editable
- I set property2 
- When I open again the ticket property2 becomes read-only

### Goals
- [ ] As a user I want properties that can be save only once 

### Tasks
- [ ] In the new ticket form show the immutable property as writable
- [ ] In the ticket details from show the immutable property as writable when they are empty and read-only once the value was set.

### Related Resources
The definition in the template data for a immutable property

```
{
  properties: [
    {
       name: "Reference number",
       type: "number",
       immutable: true
    }
   //... other properties
   ]
// ... other template values
}
```

The backend issue is #4744
